src/main/java/com/genpact/insurance/model/PolicyHolder.java

```diff
@@ -0,0 +1,34 @@
+package com.genpact.insurance.model;
+
+import java.util.Date;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Document("policyholder")
```

Comment:
make it policyholders

Reply:
have pushed the change making it policyholders